library/src/main/java/com/q42/q42stats/library/Q42Stats.kt

```diff
@@ -1,5 +1,74 @@
 package com.q42.q42stats.library
 
-class Q42Stats {
-    val name = "Q42 Stats lib"
+import android.content.Context
+import com.q42.q42stats.library.collector.AccessibilityCollector
+import com.q42.q42stats.library.collector.PreferencesCollector
+import com.q42.q42stats.library.collector.SystemCollector
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import java.io.Serializable
+import java.util.concurrent.atomic.AtomicBoolean
+
+const val TAG = "Q42Stats"
```

Comment:
If you use this in the whole app, shorter would be to set it inside the logger (so you won't have to add the TAG all the time)